bazel/container/Containerfile

```diff
@@ -8,8 +8,8 @@ ARG BAZELISK_VERSION=v1.16.0
 ARG BAZELISK_SHA256=168851e70cf5f95c0e215e7f3aaca5132ffc3c8dd8f585a4157b0be2b53cfe32
 
 ADD --checksum=sha256:${BAZELISK_SHA256} \
-    https://github.com/bazelbuild/bazelisk/releases/download/${BAZELISK_VERSION}/bazelisk-${TARGETOS}-${TARGETARCH} \
-    /usr/local/bin/bazelisk
+	https://github.com/bazelbuild/bazelisk/releases/download/${BAZELISK_VERSION}/bazelisk-${TARGETOS}-${TARGETARCH} \
+	/usr/local/bin/bazelisk
```

Comment:
Intentional change from spaces to tabs?
This file seems to only use tabs for spacing now so its fine, though our other source files use spaces instead.

Reply:
Done by my editor when i hit save. We should probably just adopt some form of auto formatting. Couldn't find a clear style guide anywhere for Dockerfiles yet.